platform/feature-api/api/pod-api/src/api.ts

```diff
@@ -131,22 +131,25 @@ export interface PolyOut extends Omit<FS, "readdir"> {
  */
 export interface PolyNav {
     /**
-     * A way for features to display a contents of a web page
+     * A way for features to display a contents of a web page of the given URL.
+     * @param {string} url - The URL to open.
      */
     openUrl(url: string): Promise<void>;
     /**
-     * Describe what actions are possible within the pod when a feature is loaded
+     * Describe what keyboard navigation actions are possible within the pod when a feature is loaded
```

Comment:
It's only keyboard navigation in pod.js, on Android/iOS there's buttons in the nav bar. The previous doc string was correct